### DISCUSSION TITLE
Feature Request: Home Page and Shelter view

### DISCUSSION THREAD

**Discussion by Pletoo** [sheltermanager/asm3#1508] [Ideas]
I would like the option to set the current home page as viewable only by specific viewer roles. Everyone else would have shelter view by default. This would allow Super Users to keep access to all the alerts and summary info without making that info available to all Normal Users. 

I would also like a Shelter view sort for most recently changed that can be set as a default.

For example...we have one volunteer who is specifically in charge of taking pictures of the new intakes. He uses the desktop/tablet interface and doesn't need to see all the alerts, recently deceased, financials...so I have cleared out the home page except for the 25 most recently changed. This makes it really easy for him, but I am missing the extra info I used to see.

**Comment by bobintetley** [comment]
Thanks for the feedback. The alerts do check permissions, so alerts do not appear if a person does not have permission to that area of the system. Eg: Without the "View Vaccination" permission, no vaccination alerts will appear, etc. It sounds like if your user in this scenario only had the permission to add media and view animal records, they shouldn't see any of the alerts and shelter view is a click away.

A "recently changed" shelterview mode could be doable, using the last changed date as the heading, I will add a ticket for that.

**Reply by Pletoo** [reply]
My test account is currently set to only view animals and add/view media. When I first was testing out what a user role sees, I was still seeing the timeline but that has been resolved. I also realized I had forgotten to set the location filter to shelter and therefore I was still seeing new fosters and recently deceased. That cleared up most of my concerns with every user seeing the home page. 

However the alerts for unvaccinated still show up. For a user that is unaware that most of these unvaccinated cats are kittens or nursing mothers in foster, it looks very alarming and a screenshot out of context could be bad. 
![image](https://github.com/user-attachments/assets/39719170-39b6-4010-8bc9-204fdf4fc207)

As you can see, the alert for long term is also visible. Our shelter is a no-kill shelter so that alert is basically just a 2nd reminder that booster shots will be due soon.

Since both of these links are clickable, they can then see a list of animals and view their records, including those in foster and permanent foster (though people names are hidden as expected.)  I am not sure if I mind the long term alert being visible to all because maybe that will help get those cats adopted, but most users should only see the cats at the shelter.

Thanks

**Reply by Pletoo** [reply]
Additionally - when I set the location filter for my test user to "Fostered Animals" I see these alerts:
![image](https://github.com/user-attachments/assets/ab575d06-0fb7-4af0-a253-58910bf55e5c)

Also, thumbnails of recently deceased foster cats show up with this setting (setting the location filter to "shelter" does not show deceased.) I have not tested this with the "My fosters" location filter.

**Comment by bobintetley** [comment]
I agree that all vacc alerts should be hidden from people who don't have view vacc permissions, even if they're related to animals never receiving a vacc. The never vaccinated alerts link to a custom search, which must not be checking the location filter (I will check this). Even so, the user cannot open records that are not in their location filter.
